lms/envs/production.py

```diff
@@ -568,6 +568,9 @@ def get_env_setting(setting):
 )
 TRACKING_SEGMENTIO_SOURCE_MAP = ENV_TOKENS.get("TRACKING_SEGMENTIO_SOURCE_MAP", TRACKING_SEGMENTIO_SOURCE_MAP)
 
+# Heartbeat
+HEARTBEAT_CELERY_ROUTING_KEY = ENV_TOKENS.get('HEARTBEAT_CELERY_ROUTING_KEY', HIGH_PRIORITY_QUEUE)
```

Comment:
Can you update the default value here to be `HEARTBEAT_CELERY_ROUTING_KEY`?  This way, (1) the value of `HIGH_PRIORITY_QUEUE` can be set only in a single place in `common.py` and (2) it supports other overrides along the way, like in `private.py`.

This will hopefully be the last change before merge.

Reply:
Sure @nasthagiri, that's a good application of DRY and I've updated this for the lms and cms `production.py`.